src/libraries/icubmod/embObjMotionControl/eomcParser.cpp

```diff
@@ -2457,6 +2457,110 @@ bool Parser::parseImpedanceGroup(yarp::os::Searchable &config,std::vector<impeda
 
 }
 
+/*
+typedef struct
+{
+    double Km;
+    double Kw;
+    double S0; 
+    double S1;
+    double Vth;
+    double Fc_pos;
+    double Fc_neg;
+    double Fs_pos;
+    double Fs_neg;
+    double tbd0;
+    double tbd1;
+    double tbd2;
+} lugreParameters_t;
+*/
+
+bool Parser::parseLugreGroup(yarp::os::Searchable &config,std::vector<lugreParameters_t> &lugre)
+{
+    Bottle lugreGroup;
+    lugreGroup=config.findGroup("LUGRE","LUGRE parameters");
+
+    if(lugreGroup.isNull())
```

Comment:
Are you sure you want to make the Lugre param mandatory?

Reply:
Thanks @valegagge for spotting this.
Nope, we cannot make the presence of LuGre section mandatory.

My previous [comment](https://github.com/robotology/icub-main/pull/989#issuecomment-2435319879) goes in the same direction.